cmd/contour/serve.go

```diff
@@ -304,18 +304,20 @@ func doServe(log logrus.FieldLogger, ctx *serveContext) error {
 	g.Add(lbsw.Start)
 
 	// step 12. register an informer to watch envoy's service if we haven't been given static details.
-	if ctx.ingressStatusAddress == "" {
+	if ctx.IngressStatusAddress == "" {
 		ssw := &k8s.ServiceStatusLoadBalancerWatcher{
-			ServiceName: ctx.envoyServiceName,
+			ServiceName: ctx.EnvoyServiceName,
 			LBStatus:    lbsw.lbStatus,
 		}
-		factory := clients.NewInformerFactoryForNamespace(ctx.envoyServiceNamespace)
+		factory := clients.NewInformerFactoryForNamespace(ctx.EnvoyServiceNamespace)
 		factory.Core().V1().Services().Informer().AddEventHandler(ssw)
 		g.Add(startInformer(factory, log.WithField("context", "serviceStatusLoadBalancerWatcher")))
+		log.WithField("envoy-service-name", ctx.EnvoyServiceName).WithField("envoy-service-namespace", ctx.EnvoyServiceNamespace).Info("Watching Service for Ingress status")
 
 	} else {
-		log.Infof("setting Ingress status to %q, disabling watching %q service", ctx.ingressStatusAddress, ctx.envoyServiceName)
-		lbsw.lbStatus <- parseStatusFlag(ctx.ingressStatusAddress)
+		log.WithField("ingress-status-address", ctx.IngressStatusAddress).Info("Using supplied information for Ingress status")
+		log.Infof("setting Ingress status to %q, disabling watching %q service", ctx.IngressStatusAddress, ctx.EnvoyServiceName)
```

Comment:
Can we collapse these into a single log message, or omit the second one?

Reply:
Oops, I had intended to remove the second one.